### ISSUE TITLE
Selenium download failing on Windows

### ISSUE THREAD

**Issue by orlade** [xolvio/meteor-cucumber#136]
I just ran a `meteor update` and was given `xolvio:cucumber` version `0.9.1`, which was surprising since it's not released on GitHub yet. In any case, I tried running my tests, and got the error:

```
W20150612-18:16:08.416(10)? (STDERR) Error: Could not request headers from http://selenium-release.storage.googleapis.com/2.45/IEDriverServer_Win32_2.45.zip:
W20150612-18:16:08.416(10)? (STDERR)     at Object.Future.wait (C:\Users\Oliver\AppData\Local\.meteor\packages\velocity_meteor-tool\1.1.3_4\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\fibers\future.js:398:15)
W20150612-18:16:08.416(10)? (STDERR)     at C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\packages\meteor.js:174:24
W20150612-18:16:08.416(10)? (STDERR)     at DEBUG (C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\packages\xolvio_cucumber.js:56:49)
W20150612-18:16:08.416(10)? (STDERR)     at DEBUG (C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\packages\xolvio_cucumber.js:125:3)
W20150612-18:16:08.416(10)? (STDERR)     at C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\packages\xolvio_cucumber.js:129:4
W20150612-18:16:08.417(10)? (STDERR)     at C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\packages\xolvio_cucumber.js:554:3
W20150612-18:16:08.417(10)? (STDERR)     at C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\boot.js:222:10
W20150612-18:16:08.417(10)? (STDERR)     at Function._.each._.forEach (C:\dev\workspace\fabric\node_modules\underscore\underscore.js:153:9)
W20150612-18:16:08.417(10)? (STDERR)     at C:\dev\workspace\fabric\app\.meteor\local\build\programs\server\boot.js:117:5
W20150612-18:16:08.417(10)? (STDERR)     - - - - -
W20150612-18:16:08.417(10)? (STDERR)     at Request.<anonymous> (C:\Users\Oliver\AppData\Local\.meteor\packages\xolvio_cucumber\0.9.1\npm\node_modules\chimp\node_modules\selenium-standalone\lib\install.js:280:17)
W20150612-18:16:08.417(10)? (STDERR)     at Request.emit (events.js:95:17)
W20150612-18:16:08.418(10)? (STDERR)     at Request.onRequestResponse (C:\Users\Oliver\AppData\Local\.meteor\packages\xolvio_cucumber\0.9.1\npm\node_modules\chimp\node_modules\request\request.js:1156:10)
W20150612-18:16:08.418(10)? (STDERR)     at ClientRequest.emit (events.js:95:17)
W20150612-18:16:08.418(10)? (STDERR)     at HTTPParser.parserOnIncomingClient (http.js:1693:21)
W20150612-18:16:08.418(10)? (STDERR)     at HTTPParser.parserOnHeadersComplete [as onHeadersComplete] (http.js:122:23)
W20150612-18:16:08.418(10)? (STDERR)     at Socket.socketOnData (http.js:1588:20)
W20150612-18:16:08.418(10)? (STDERR)     at TCP.onread (net.js:528:27)
```

`http://selenium-release.storage.googleapis.com/2.45/IEDriverServer_Win32_2.45.zip` doesn't exist. This error does not occur in `0.8.6`.

**Comment by samhatoum** [comment]
Curious! The real URL is here:

https://selenium-release.storage.googleapis.com/2.45/IEDriverServer_Win32_2.45.0.zip

The one above is missing a `.0`. I haven't updated the versions. If it worked before then perhaps the URLs changed? I'll bump the versions and see if that helps. I'll shortly release `0.9.2` and I would appreciate it if you can give me some feedback as I don't have a windows machine to hand.

**Comment by orlade** [comment]
Will do. It seemed to break from 0.8.6 to 0.9.x, so I'll check out 0.9.2.

**Comment by samhatoum** [comment]
Actually, would you try this first:

Could you on windows do"

```
npm install -g selenium-standalone

selenium-standalone start --drivers.ie.version=2.46 --drivers.ie.baseURL=http://selenium-release.storage.googleapis.com
```

If that doesn't work, then could you try:

```
selenium-standalone start --drivers.ie.version=2.46.0 --drivers.ie.baseURL=http://selenium-release.storage.googleapis.com
```

Let me know, in the meantime I'm bumping all other versions of everything!

**Comment by orlade** [comment]
Sorry, I didn't get a chance to test that before `0.9.2` landed, but I can at least confirm that `0.9.2` works. Still, I can run them anyway.

First command:

```
λ selenium-standalone start --drivers.ie.version=2.46 --drivers.ie.baseURL=http://selenium-release.storage.googleapis.com
C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\bin\selenium-standalone:54
        throw err;
              ^
Error: Missing C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\iedriver\2.46-x64-IEDriverServer.exe
    at C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\lib\check-paths-existence.js:15:20
    at FSReqWrap.cb [as oncomplete] (fs.js:226:19)
```

It says there's a file missing, so I tried `selenium-standalone install` instead:

```
λ selenium-standalone install --drivers.ie.version=2.46 --drivers.ie.baseURL=http://selenium-release.storage.googleapis.com
----------
selenium-standalone installation starting
----------

---
selenium install:
from: http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar
to: C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\selenium-server\2.45.0-server.jar
---
chrome install:
from: http://chromedriver.storage.googleapis.com/2.15/chromedriver_win32.zip
to: C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\chromedriver\2.15-x64-chromedriver
---
ie install:
from: http://selenium-release.storage.googleapis.com/2.45/IEDriverServer_x64_2.46.zip
to: C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\iedriver\2.46-x64-IEDriverServer.exe
---
File from http://chromedriver.storage.googleapis.com/2.15/chromedriver_win32.zip has already been downloaded
---
File from http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar has already been downloaded
C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\bin\selenium-standalone:86
        throw err;
              ^
Error: Could not request headers from http://selenium-release.storage.googleapis.com/2.45/IEDriverServer_x64_2.46.zip:
    at Request.<anonymous> (C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\lib\install.js:280:17)
    at Request.emit (events.js:107:17)
    at Request.onRequestResponse (C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\node_modules\request\request.js:1142:10)
    at ClientRequest.emit (events.js:107:17)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:426:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:111:23)
    at Socket.socketOnData (_http_client.js:317:20)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
```

Didn't work, so I tried the second command:

```
λ selenium-standalone install --drivers.ie.version=2.46.0 --drivers.ie.baseURL=http://selenium-release.storage.googleapis.com
----------
selenium-standalone installation starting
----------

---
selenium install:
from: http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar
to: C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\selenium-server\2.45.0-server.jar
---
chrome install:
from: http://chromedriver.storage.googleapis.com/2.15/chromedriver_win32.zip
to: C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\chromedriver\2.15-x64-chromedriver
---
ie install:
from: http://selenium-release.storage.googleapis.com/2.45/IEDriverServer_x64_2.46.0.zip
to: C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\.selenium\iedriver\2.46.0-x64-IEDriverServer.exe
---
File from http://chromedriver.storage.googleapis.com/2.15/chromedriver_win32.zip has already been downloaded
---
File from http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar has already been downloaded
C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\bin\selenium-standalone:86
        throw err;
              ^
Error: Could not request headers from http://selenium-release.storage.googleapis.com/2.45/IEDriverServer_x64_2.46.0.zip:
    at Request.<anonymous> (C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\lib\install.js:280:17)
    at Request.emit (events.js:107:17)
    at Request.onRequestResponse (C:\Users\Oliver\AppData\Roaming\npm\node_modules\selenium-standalone\node_modules\request\request.js:1142:10)
    at ClientRequest.emit (events.js:107:17)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:426:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:111:23)
    at Socket.socketOnData (_http_client.js:317:20)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
```

I have no idea what I'm doing with Selenium, but hopefully that tells you something :).
